README.md

```diff
@@ -85,7 +85,7 @@ The gateway UI container is available at [`fedimintui/gateway-ui`](https://hub.d
 docker pull fedimintui/gateway-ui:0.1.0
 docker run \
   --platform linux/amd64 \
-  --env "REACT_APP_FM_GATEWAY_API='ws://127.0.0.1:8175'" \
+  --env "REACT_APP_FM_GATEWAY_API='http://127.0.0.1:8175'" \
   --env REACT_APP_FM_GATEWAY_PASSWORD=password \
```

Comment:
can we set all these gateway passwords to "thereisnosecondbest" to keep in line with the fedimint docker images and setup guides we normally use?

Reply:
These are instructions for running a production release, so I would definitely want to avoid encouraging someone to continue to use that password on a real instance. It's meant to be replaced, not run as-is, so maybe we should make it something obvious like `[your-password-here]`

Reply:
+1 to @wbobeirne suggestion, and I'd follow up with these as doc updates given they're going to conflict with #274

Reply:
Dropped this tangential change to avoid conflicts with mentioned PR.

Reply:
#288